src/constants.cr

```diff
@@ -22,6 +22,7 @@ module PlaceOS
   USERNAME         = ENV["PLACE_USERNAME"]? || abort("missing PLACE_USERNAME")
   PASSWORD         = ENV["PLACE_PASSWORD"]? || abort("missing PLACE_PASSWORD")
   AUTH_HOST        = ENV["PLACE_AUTH_HOST"]? || "auth"
+  KIBANA_ROUTE     = ENV["KIBANA_ROUTE"]? || "monitor"
```

Comment:
do you think we should prepend with `PLACE_`? just to make sure it's obvious that the env is placeos related?

Reply:
Yeah good idea. I'll also change it to `PLACE_METRICS_ROUTE` since it doesn't have to be kibana